docker-compose.yml

```diff
@@ -9,6 +9,11 @@ services:
     - .:/var/www/html # Non Mac users.
     # - nfsmount:/var/www/html # Mac Users with the nfsmount volume.
     environment:
+      # ASDA CREDENTIALS
```

Comment:
Make sure this modification is also include in `drupal-site-template` docker file.

Reply:
@voidtek @jonhy81 I was thinking, shouldn't we namespace all Toolkit related env variable with `TOOLKIT_`? so like:

```
        asda-user: ${env.TOOLKIT_ASDA_USER}
        asda-password: ${env.TOOLKIT_ASDA_PASSWORD}
```

Reply:
Sounds good to me, but ASDA can be used without toolkit, in fact a developer can maintain their own snippet to download it for some local test.

Reply:
@ademarco ASDA_USER and ASDA_PASSWORD is not for toolkit but for ASDA, so for me, the naming is good.